coverage/main.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/coverage/artifacts"
+	"os"
+)
+
+const (
+	keyCovProfileFileName    = "key-cov-prof.txt"
+	defaultStdoutRedirect    = "stdout.txt"
+	defaultCoverageTargetDir = "./pkg/"
+	defaultGcsBucket         = "knative-prow"
+	defaultPostSubmitJobName = "post-knative-serving-go-coverage"
+	defaultCovThreshold      = 80
+)
+
+var (
+	artifactsDir        = "./artifacts/"
+	coverageProfileName = "coverage_profile.txt"
+)
+
+func main() {
+	fmt.Println("entering code coverage main")
```

Comment:
Why not using `logrus` here, in line 65 and 78? You'll at least get consistent logging.

Reply:
I was thinking about making these messages standing out, as they mark the beginning and ending of the binary run. If everything is in more format it's kind of hard to locate these ones.

Reply:
Why would they stand out? If you want to locate them quickly, you can add some decoration.

Reply:
To make a message stand out with log formatting either:
- use a different log level (eg `logrus.Debug`)
- add another field to the log output. (eg `logrus.WithFields(...).Info(...)`)

but uh, aside from that, is this really a useful log message to begin with? "program is running" doesn't seem helpful.